homeassistant/components/flume/__init__.py

```diff
@@ -81,3 +103,29 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         hass.data[DOMAIN].pop(entry.entry_id)
 
     return unload_ok
+
+
+async def async_setup_service(hass: HomeAssistant) -> None:
+    """Add the services for the flume integration."""
+
+    async def list_notifications(call: ServiceCall) -> ServiceResponse:
```

Comment:
Same here.

Reply:
Done in https://github.com/home-assistant/core/pull/102807